pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -427,7 +427,7 @@ public void close() throws IOException {
     public void unloadNamespaceBundlesGracefully() {
         try {
             // make broker-node unavailable from the cluster
-            if (pulsar.getLoadManager() != null) {
+            if (pulsar.getLoadManager() != null && pulsar.getLoadManager().get() != null) {
```

Comment:
this is an unrelated change. can you remove it?

Reply:
Yeah, I removed it. :)